great-idea/css/index.css

```diff
@@ -49,18 +130,245 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+html,
+body {
+	height: 100%;
+	font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: "Bangers", cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+}
+
+body {
+	line-height: 1;
+}
+ol,
+ul {
+	list-style: none;
+}
+blockquote,
+q {
+	quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+	content: "";
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+	box-sizing: border-box;
+}
+
+html,
+body {
+	height: 100%;
+	font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: "Bangers", cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+* {
+	box-sizing: border-box;
+	padding: 10;
+	margin: 0;
+	border: 1px solid grey;
+}
+
+header nav {
+	display: flex;
+	justify-content: space-evenly;
+	color: lightgray;
+	margin: 35px 60px 60px;
+	text-align: center;
+}
+nav a {
+	text-decoration: none;
+	color: grey;
+	font-size: 13px;
+	letter-spacing: 1.5px;
+	margin-left: 10px;
+	margin-right: 10px;
+}
+nav a:hover {
+	/*styles are applied when a user hovers over this element*/
+	color: #4682b4;
+	text-decoration: overline;
+}
+
+.top-page {
+	display: flex;
```

Comment:
flex grow is not necessary.  you could try taking away the padding and margin-bottom and change justify-content to space-evenly to put an even amount of space around the two flex items.  Then margin of maybe 60px all the way around would push everything down from the header and push the flex items in .top-page in a little.  Align items center will center both flex items vertically.